commerce_hubspot.services.yml

```diff
@@ -0,0 +1,7 @@
+services:
+  commerce_hubspot.sync_to.service:
```

Comment:
**CHANGE REQUEST**

The service suffix is not needed on the service ID. It is a service, it's obvious. Change this to `commerce_hubspot.sync_to`. In the class name we do add `Service` suffix i.e. `SyncToService` because it is a PHP class, by looking at the file it's not obvious at all that this is a service made available by the service container. Keep the class name as is.